manifests/dynflow/worker.pp

```diff
@@ -11,15 +11,15 @@
   Boolean $service_enable = $::foreman::jobs_service_enable
 ) {
 
+  Class['foreman::database::postgresql'] ~>
```

Comment:
Not sure if this is correct way to handle. The issue is that a worker can involve connecting to the database and therefore shouldn't be started until database setup has completed.